libs/sdk-ui-vis-commons/src/compactSize/HeadlinePagination.tsx

```diff
@@ -1,13 +1,19 @@
 // (C) 2021 GoodData Corporation
 import React, { useState } from "react";
 
-import { HeadlinePaginationRenderer } from "./HeadlingePaginationRenderer";
+import { HeadlinePaginationRenderer } from "./HeadlinePaginationRenderer";
 
-interface IHeadlinePaginationProps {
+/**
+ * @public
```

Comment:
This should be `@internal` since it is in the `vis-commons` package.

Reply:
Changed all to `@internal`.